packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -856,7 +856,7 @@ export const components = ({
             paddingLeft: odysseyTokens.Spacing4,
           },
 
-          "&.isClickable:hover": {
+          "&:hover": {
```

Comment:
This we probably need to keep. You didn't add this did you? I think removing this here breaks other implementations